ground/gcs/src/app/main.cpp

```diff
@@ -264,6 +265,9 @@ int main(int argc, char **argv)
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
     }
+    QProcess process;
```

Comment:
Could you add a comment as to what this code is doing? It's relatively opaque what the problem is and where the solution is coming from. Is this inspired from a SO page which could be referred to?